.github/workflows/publish.yml

```
@@ -18,3 +18,8 @@ jobs:
           identifier: sxyazi.yazi
           installers-regex: 'yazi-(x86_64|aarch64)-pc-windows-msvc\.zip$'
           token: ${{ secrets.WINGET_TOKEN }}
+
+      - name: Promote snap to latest/stable
+        env:
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_TOKEN }}
+        run: snapcraft promote -v --from-channel latest/candidate --to-channel latest/stable --yes
```

It seems that `snapcraft promote` does not have `-v` and the package name is missing

```suggestion
        run: snapcraft promote yazi --from-channel latest/candidate --to-channel latest/stable --yes
```

---

When I run `snapcraft promote yazi -v --from-channel latest/candidate --to-channel latest/stable --yes` (with `-v`), got:

```
❯ snapcraft promote yazi -v --from-channel latest/candidate --to-channel latest/stable --yes
Starting snapcraft, version 8.6.3
Logging execution to '/Users/ika/Library/Logs/snapcraft/snapcraft-20250304-145249.730502.log'
Usage: snapcraft promote [OPTIONS] <snap-name>
Try 'snapcraft promote -h' for help.

Error: No such option: -v
```

But strangely, in `snapcraft promote -h` I can see the `-v` option. Is this a bug?

---

I think that might be a bug, I'll report separately but have fixed in the latest commit anyway.